permissions/plugin-pipeline-config.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-model-definition"
+paths:
+- "org/jenkinsci/plugins/pipeline-model-definition"
```

Comment:
no strong opinion. We have no standard from this, and plugin devs use whatever they want